### PR TITLE
[EWL-4630] - Add use cases

### PR DESCRIPTION
**Pull request by nstriedinger**
## Ticket(s)

**Jira Ticket**
- https://issues.ama-assn.org/browse/EWL-4630

## Open Questions
- Three patterns were not mentioned in the spreadsheet. 'Partners List', 'Masthead', and 'Sidebar Article' -- Should they have `Use Cases`?

## Description
Add use cases for  `Pattern Info` from:
https://docs.google.com/spreadsheets/d/1MROQSqi-yqqchAMx_iwU3t3Retu5aA8jbJU-XLxlfYg/edit#gid=1072455826

## To Test
- [ ] Open spreadsheet and verify **Molecules** and **Organisms** have the correct `Use Cases`.

## Visual Regressions
N/A

## Relevant Screenshots/GIFs
N/A

## Remaining Tasks
N/A

## Additional Notes
N/A

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

**Comment by designerneil** [comment]
@nstriedinger I noticed there is no use case for atoms ie. buttons, copyright, date block, links was there a reason why these were omitted?

**Comment by nstriedinger** [comment]
@designerneil -- We aren't doing use cases for 'Atoms'

**Review by designerneil — APPROVED** [review]

File: styleguide/source/_patterns/02-molecules/press-release-logos.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ A image parade used specifically for the press page.
 
 [EWL-4486](https://issues.ama-assn.org/browse/EWL-4486)
 
+### Use Case
+
```
Comment (labbydev) [inline styleguide/source/_patterns/02-molecules/press-release-logos.md]: If there's no content for this we should go ahead and remove it here.

File: styleguide/source/_patterns/03-organisms/masthead.md [hunk 2]
```diff
@@ -11,6 +11,8 @@ The Masthead includes a People Bio, a group of social icons/links, a date, a cat
 ### Variants
 - [Masthead with Subtitle](?p=organisms-masthead-with-subtitle)
 
+### Use Case
```
Comment (labbydev) [inline styleguide/source/_patterns/03-organisms/masthead.md]: If there's no content for this we should go ahead and remove it here.

File: styleguide/source/_patterns/03-organisms/partner-list.md [hunk 3]
```diff
@@ -7,6 +7,8 @@ A collection of up to three articles displayed in a column (meant for use in a s
 
 [EWL-4433](https://issues.ama-assn.org/browse/EWL-4434)
 
+### Use Case
```
Comment (labbydev) [inline styleguide/source/_patterns/03-organisms/partner-list.md:10]: If there's no content for this we should go ahead and remove it here.

File: styleguide/source/_patterns/03-organisms/sidebar-articles.md [hunk 4]
```diff
@@ -8,6 +8,7 @@ This pattern is a block that may optionally include:
 
 [EWL-4389](https://issues.ama-assn.org/browse/EWL-4389)
 
+### Use Case
```
Comment (labbydev) [inline styleguide/source/_patterns/03-organisms/sidebar-articles.md:11]: If there's no content for this we should go ahead and remove it here.

**Review by labbydev — APPROVED** [review]

**Comment by labbydev** [comment]
@nstriedinger things look good but I would remove the blank use cases. 

I'd like to get @sarahdlowe to look at this as well.

**Comment by sarahdlowe** [comment]
Thanks @labbydev I'm going to take a look at this by the end of this week.

**Comment by sarahdlowe** [comment]
@nstriedinger I have gone through the spreadsheet and updated the Business Use Case column for  **Molecules** and **Organisms** and added use cases for the three patterns 'Partners List', 'Masthead', and 'Sidebar Article'

**Pull request merged**